test/versioned/otel-bridge/span.test.js

```diff
@@ -673,6 +673,38 @@ test('server span should not accept upstream traceparent/tracestate if distribut
   })
 })
 
+test('Span errors are handled and added on transaction', (t, end) => {
+  const { agent, tracer } = t.nr
+  helper.runInTransaction(agent, (tx) => {
+    tx.name = 'http-external-test'
+    tracer.startActiveSpan('http-outbound', { kind: otel.SpanKind.CLIENT }, (span) => {
```

Comment:
can you also add a test that asserts no errors getting added when `span.status_code` is not 2?